### PR TITLE
Fix path to initdb and weird Ubuntu unit

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: tasks/install.yml [hunk 1]
```diff
@@ -72,3 +97,15 @@
...
     - "{{ postgres_home_directory }}"
+
+- name: Set path to postgres utilities
+  set_fact:
+    "{{ item.name }}": "{% if pg_ver.stdout %}/usr/lib/postgresql/{{ pg_ver.stdout }}/bin/{{ item.value }}{% else %}{{ item.defalt }}{% endif %}"
+  when: "{{ item.name }} is not defined"
+  with_items:
+    - name: postgres_initdb
+      value: initdb
+      default: postgres_default_initdb
```
Comment (SlothOfAnarchy) [inline tasks/install.yml]: nice, I didn't know this way of writing this.

File: tasks/install.yml [hunk 2]
```diff
@@ -72,3 +97,15 @@
     - "{{ postgres_data_directory }}"
     - "{{ postgres_log_directory }}"
     - "{{ postgres_home_directory }}"
+
+- name: Set path to postgres utilities
+  set_fact:
+    "{{ item.name }}": "{% if pg_ver.stdout %}/usr/lib/postgresql/{{ pg_ver.stdout }}/bin/{{ item.value }}{% else %}{{ item.defalt }}{% endif %}"
```
Comment (SlothOfAnarchy) [inline tasks/install.yml]: defalt -> default

**Review by SlothOfAnarchy — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
@SlothOfAnarchy Pls rereview

**Review by SlothOfAnarchy — APPROVED** [review]
Ok, but please don't force-push next time.

**Pull request merged**